interfaces/acados_template/acados_template/c_templates/acados_solver_sfun.in.c

```diff
@@ -105,7 +105,7 @@ static void mdlInitializeSizes (SimStruct *S)
     // specify the direct feedthrough status 
     ssSetInputPortDirectFeedThrough(S, 0, 1); // current state x0
     ssSetInputPortDirectFeedThrough(S, 1, 1); // y_ref
-    ssSetInputPortDirectFeedThrough(S, 2, 1); // y_ref_N
+    ssSetInputPortDirectFeedThrough(S, 2, 1); // y_ref__e
```

Comment:
here the is a double underscore

Reply:
wow, good catch! It's a comment, but still.

Reply:
it just jumped into my eye